src/xpk/core/blueprint/blueprint_generator.py

```diff
@@ -354,6 +360,289 @@ def _get_a3_mega_blueprint_dependencies(
     )
     return deployment_files_path
 
+  def _get_a3_ultra_blueprint_dependencies(
+      self, blueprint_name: str, prefix: str = ""
+  ) -> str:
+    deployment_files_path = os.path.join(
+        self._get_storage_path(prefix), blueprint_name
+    )
+    shutil.copytree(
+        blueprint_dependencies_dir[a3ultra_device_type], deployment_files_path
+    )
+    return deployment_files_path
+
+  def generate_a3_ultra_blueprint(
+      self,
+      project_id: str,
+      cluster_name: str,
+      blueprint_name: str,
+      region: str,
+      zone: str,
+      auth_cidr: str,
+      extended_reservation: str | None,
+      system_node_pool_machine_type: str,
+      num_nodes: int,
+      autoscaling_total_min_nodes: int,
+      static_node_count: int = 4,
+      prefix: str = "",
+      system_node_pool_disk_size_gb: int = 200,
```

Comment:
why do we have these limits? why not to leave so that gke set them with defaults?
```
      system_node_pool_disk_size_gb: int = 200,
      a3ultra_node_pool_disk_size_gb: int = 100,
```

Reply:
values copied from ct blueprint

Reply:
> why not to leave so that gke set them with defaults?